### PR TITLE
chore: Updates Aries version

### PR DESCRIPTION
**Pull request by soluchok**
Updates Aries version and uses `agent-sdk`.

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: pkg/restapi/issuer/operation/support_test.go [hunk 1]
```diff
@@ -344,7 +344,6 @@ const (
 		"birthDate": "1958-07-17"
 	  },
 	   "metadata":{
-		  "contexts":["https://w3id.org/citizenship/v1"],
```
Comment (llorllale) [inline pkg/restapi/issuer/operation/support_test.go:347]: why is this removed?

Comment (soluchok) [reply pkg/restapi/issuer/operation/support_test.go:347]: due to error `error creating reference credential : create credential : sign user data vc : failed to sign credential: add linked data proof: failed to normalize JSON-LD document: loading remote context failed: Dereferencing a URL did not result in a valid JSON-LD context: https://w3id.org/citizenship/v1"`

Comment (soluchok) [reply pkg/restapi/issuer/operation/support_test.go:347]: https://w3id.org/citizenship/v1 URL is invalid. (loading remote context failed)

Comment (llorllale) [reply pkg/restapi/issuer/operation/support_test.go:347]: @rolsonquadras ^

Comment (soluchok) [reply pkg/restapi/issuer/operation/support_test.go:347]: I also assume this is the reason why BDD tests are failing.  https://w3id.org/citizenship/v3 also invalid.

Comment (llorllale) [reply pkg/restapi/issuer/operation/support_test.go:347]: Related: #114 #306

Comment (rolsonquadras) [reply pkg/restapi/issuer/operation/support_test.go:347]: Rather than depending on external sources, we can add the citizenship context in trustbloc/context for our tests. We would have grater control on that.

Comment (rolsonquadras) [reply pkg/restapi/issuer/operation/support_test.go:347]: Created https://github.com/trustbloc/context/issues/58.

Comment (llorllale) [reply pkg/restapi/issuer/operation/support_test.go:347]: @rolsonquadras 

> Rather than depending on external sources, we can add the citizenship context in trustbloc/context for our tests. We would have grater control on that.

-1

This context is used for interop.

Comment (rolsonquadras) [reply pkg/restapi/issuer/operation/support_test.go:347]: These are not interop tests. We will use interop contexts when we have interop tests in adapters.

Comment (llorllale) [reply pkg/restapi/issuer/operation/support_test.go:347]: @rolsonquadras 

> These are not interop tests. We will use interop contexts when we have interop tests in adapters.

This path would lead to tests using different URLs which will just confuse everyone. What is the reason for not configuring document loaders as recommended by JSONLD advocates? (#114, #306)

Comment (rolsonquadras) [reply pkg/restapi/issuer/operation/support_test.go:347]: Yes, we have to implement #114 and #306 

But to unblock this PR, I added the context.

Comment (rolsonquadras) [reply pkg/restapi/issuer/operation/support_test.go:347]: @soluchok can you replace the context url with https://trustbloc.github.io/context/vc/examples/citizenship-v1.jsonld one and check ?

Comment (llorllale) [reply pkg/restapi/issuer/operation/support_test.go:347]: @rolsonquadras 

> Yes, we have to implement #114 and #306
> 
> But to unblock this PR, I added the context.

Well then, great! Glad we agree! :)

I would never have guessed based on your prior comments.

Comment (rolsonquadras) [reply pkg/restapi/issuer/operation/support_test.go:347]: My bad, I wasn't clear enough. Since those PRs are open and that's our long term goal, I thought its not really need to mention that. :)

**Review by rolsonquadras — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

**Review by llorllale — APPROVED** [review]

**Review by sudeshrshetty — APPROVED** [review]
